server/service/orbit.go

```diff
@@ -143,10 +143,13 @@ func (svc *Service) EnrollOrbit(ctx context.Context, hostInfo fleet.OrbitHostInf
 		return "", orbitError{message: "app config load failed: " + err.Error()}
 	}
 
-	_, err = svc.ds.EnrollOrbit(ctx, appConfig.MDM.EnabledAndConfigured, hostInfo, orbitNodeKey, secret.TeamID)
+	h, err := svc.ds.EnrollOrbit(ctx, appConfig.MDM.EnabledAndConfigured, hostInfo, orbitNodeKey, secret.TeamID)
 	if err != nil {
 		return "", orbitError{message: "failed to enroll " + err.Error()}
 	}
+	if err := svc.ds.BulkSetPendingMDMAppleHostProfiles(ctx, []uint{h.ID}, nil, nil); err != nil {
```

Comment:
So instead of calling this in DEP/manual enrollment, I call it only here, in the orbit enrollment, as I believe this is required anyway for MDM deployments, so it is one place where the host will enroll regardless of DEP/manual. However, maybe it should _also_ be called in DEP/manual when a host enrolls in orbit first and in mdm next?

Reply:
but... at the same time, I don't know how critical it is to set all profiles as pending during enrollment. I mean it's ideal, but the cron will eventually catch up.